torch/csrc/autograd/profiler.h

```diff
@@ -207,6 +288,9 @@ struct TORCH_API Event final {
   int64_t cuda_memory_usage_ = 0;
   int device_ = -1;
   struct CUevent_st* cuda_event = nullptr;
+  int node_id_ = 0;
+  bool is_remote_ = false;
+  double cuda_us_ = -1;
```

Comment:
why is it double? maybe int64_t like cpu_ns_?

Reply:
We compute this as 
```
double cuda_us =
                        cuda_elapsed_us + cudaProfilerStartEvent->cpu_us();
                    e.setCudaUs(cuda_us);
```

`cuda_elapsed_us` is the result of `cuda_elapsed_us()` function, which returns a double, so the sum should be double as well? Unless cuda_elapsed_us() should also return an int64 or it's okay to convert double -> int64 here.

Reply:
cuda_elapsed_us is cudaEventElapsedTime multiplied by 1000.0;
checked nvidia docs:
"Computes the elapsed time between two events (in milliseconds with a resolution of around 0.5 microseconds)"
I guess it's ok to convert to int64_t to fully mimic cpu_ns_

Reply:
Sounds good, will update